### ISSUE TITLE
Can not run on Mac OS

### ISSUE THREAD

**Issue by alexnkorovin** [PetervanLunteren/EcoAssist#21]
Hello,

I have installed EcoAssist on Max with Intel CPU with MacOS Ventura 13.4.1 (c) (22F770820d) and Xcode 14.3.1 (14E300c)
Installation has finished without any visible issues, however when I double clicking EcoAssist.command file the Terminal window appears with the following text:

/Applications/EcoAssist.command ; exit;
(base) ➜  ~ /Applications/EcoAssist.command ; exit;
This is an OSX computer...
   ...with an Intel processor.

Saving session...completed.

[Process completed]

No application window is opened. Could you give a hint for this issue.
I really like the idea and would like to test you package

Thank you for the help!

**Comment by PetervanLunteren** [comment]
Hi @alexnkorovin,

Not sure what's going on, but we can find out! You can enable console output by

1. opening your `/Applications/EcoAssist.command` with a text editor like TextEdit or Visual Studio Code;
2. outcomment [these lines](https://github.com/PetervanLunteren/EcoAssist/blob/main/open.command#L35-L36) like so:
```bash
# exec 1> $LOCATION_ECOASSIST_FILES/EcoAssist/logfiles/stdout.txt
# exec 2> $LOCATION_ECOASSIST_FILES/EcoAssist/logfiles/stderr.txt
```
3. close and save the file;
4. start EcoAssist as you would do normally.

This will not fix the issue, but it will give us more information in whats going on. Please share the console output here.

Thanks!

**Comment by alexnkorovin** [comment]
Thank you for your hint!
Problem with starting up the app was solved.

I checked up stderr.txt file and found the error massage:
  stderr: 'xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun'

Looking around I found solution here: https://setapp.com/how-to/fix-xcrun-error
The issue is in missing xcode path after upgrading to later version of MacOS.
Problem was solver with the terminal command: sudo xcode-select -s /Applications/Xcode.app
